amplify/src/main/java/com/github/stkent/amplify/tracking/predicates/LastTimePredicate.java

```diff
@@ -40,4 +40,9 @@ public void eventTriggered(@NonNull IEvent event) {
             updateEventValue(event, currentTime);
         }
     }
+
+    @Override
+    public Long defaultValue() {
+        return 0L;
```

Comment:
Yup, any other ideas here? We don't want to tie the checks to the predicates or events.